packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -21,13 +21,14 @@ These limits are part of the Okta [Rate limit](/docs/reference/rate-limits) poli
 > **Note:**
 >
 > * In addition to the rate limits listed on this page, Okta applies rate limits per API, divided into three categories. See the [Rate limit overview](/docs/reference/rate-limits).
+> * [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/) apply to a variety of endpoints across different APIs for customers that purchased this add-on.
```

Comment:
Should this be present tense?

Reply:
This should be OK as is. Our customers must have previously purchased the add-on before they can apply dynamic scale rate limits.